### ISSUE TITLE
[BUG] ImportTnCheckerPayload has missing SiteId and SipPeerId parameters, and produces XML that doesn't conform API documentation

### ISSUE THREAD

**Issue by kolygin** [Bandwidth/csharp-bandwidth-iris#68]
### Checklist
 - [X] Have you provided a description of the bug?
 - [X] Have you provided your Environment information?
 - [X] Have you provided a sample code snippet?
 - [ ] Have you provided a stack trace?
 - [X] Have you outlined the expected behavior?

### Description
ImportTnChecker model do not match API documentation. As a result, 400 Bad Request is returned by api/accounts/{accountId}/importTnChecker API endpoint.
Specifically, there are two issues:
1. SiteId and SipPeerId parameters are missing in ImportTnCheckerPayload class
2. TelephoneNumbers array is being serialized in a way that doesn't match API documentation. This in turn prevents API from recognizing numbers client is sending.

### Environment Information
 - OS Version: Windows 10
 - SDK Version: 3.1.0
 - Environment: .NET Core 6.0

### Sample Code Snippet

```csharp
            var checkerRequest = new ImportTnCheckerPayload
            {
                TelephoneNumbers = new TelephoneNumber[]
                {
                    new TelephoneNumber
                    {
                        FullNumber = "5105551122"
                    }
                },
            };

            var xmlSerializer = new XmlSerializer(checkerRequest.GetType());
            using Utf8StringWriter writer = new Utf8StringWriter();
            xmlSerializer.Serialize(writer, checkerRequest);
            string content = writer.ToString();
```

### Expected Behavior
API documentation at https://dev.bandwidth.com/docs/numbers/guides/hostingNumbers/ suggests that expected request format is:
```xml
<ImportTnCheckerPayload>
    <TelephoneNumbers>
        <TelephoneNumber>3032281000</TelephoneNumber>
        <TelephoneNumber>4109235436</TelephoneNumber>
        <TelephoneNumber>4104685864</TelephoneNumber>
    </TelephoneNumbers>
    <SiteId>486</SiteId>
    <SipPeerId>500025</SipPeerId>
</ImportTnCheckerPayload>
```

### Actual behavior
The payload being produced by XmlSerializer off the defined model classes is:
```xml
<ImportTnCheckerPayload>
  <TelephoneNumbers>
    <TelephoneNumber>
      <FullNumber>5105551122</FullNumber>
    </TelephoneNumber>
  </TelephoneNumbers>
</ImportTnCheckerPayload>
```
Notice extra FullNumber element.

### Suggested Fix

1. Fix XML serialization hints for TelephoneNumbers .
2. Add SiteId and SipPeerId properties

```csharp
    public class ImportTnCheckerPayload
    {
        public string SiteId { get; set; } 
        public string SipPeerId { get; set; }
        [XmlArray("TelephoneNumbers")]
        [XmlArrayItem("TelephoneNumber")]
        public string[] TelephoneNumbers { get; set; }

        [XmlArray("ImportTnErrors")]
        [XmlArrayItem("ImportTnError")]
        public ImportTnError[] ImportTnErrors { get; set; }
    }
```

**Comment by ckoegel** [comment]
Thanks for the heads up @kolygin, we'll look into this asap and hopefully roll out the fix for the next release on August 30th.

**Comment by kolygin** [comment]
@ckoegel Please bear in mind that ImportTnError class has in turn its own array of TelephoneNumbers which might be also affected by this problem.

**Comment by kolygin** [comment]
@ckoegel Also, ImportTnOrder model class and its children may have the same issue.
1. TelephoneNumbers -> TelephoneNumber -> FullNumber
2. Subscriber.Name (doc) vs Subscriber.FirstName + LastName (model) mismatch
3. Errors array not defined in ImportTnOrder

**Comment by matthewkmartin** [comment]
Closed by #69
